package.json

```diff
@@ -181,7 +181,7 @@
     "serve-notify": "export NOTIFY=true && npm run serve",
     "scaffold": "./node_modules/.bin/gulp ensureConfig && ./node_modules/.bin/gulp ensureDevProxy",
     "prestart": "npm run check-env",
-    "start": "NODE_ENV=development webpack-dashboard -- webpack-dev-server --config ./webpack/webpack.dev.js --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
+    "start": "NODE_ENV=development webpack-dashboard -m -- webpack-dev-server --config ./webpack/webpack.dev.js --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
```

Comment:
Sneaking the -m because it helps a lot to see the dashboard. Would just drop the commit if someone does not like it.

Reply:
Although I would love to see a seperate PR (and therefore also ticket) for this I like the change 👍

Reply:
I feel you, and I understand it in the context of the discussion we had on sync. I just took a risk here, if someone opposed I would drop the commit.

Reply:
I see you are having the same issues I'm too but having to add `-m` is only targeting the symptom and not the root cause. I would rather see us removing `webpack-dashboard` entirely https://github.com/dcos/dcos-ui/pull/2892